### DISCUSSION TITLE
How to async load each language in v3

### DISCUSSION THREAD

**Discussion by joaopedrodcf** [js-fns/date-fns#3663] [Q&A]
Hey @kossnocorp,

First thanks for this awesome library 🥇 

We are using it on version 2 currently and would like to migrate to version 3, and one of the optimisations we really like about version 2 was being able to async load the locales:

```ts
await import(`date-fns/locale/${locale}/index.js`)
```

![image](https://github.com/date-fns/date-fns/assets/11476152/a7e99be1-26c7-41d2-bf51-826c6fa8e43a)

But now with v3 this does not work, any ideas ?

**Comment by jeanpierrecarvalho** [comment]
@joaopedrodcf this will be helpful! Thanks a lot :rocket:
